src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -66,6 +66,14 @@ export default class GooglePayCustomerStrategy implements CustomerStrategy {
         );
     }
 
+    executePaymentProviderCheckout(options?: ExecutePaymentProviderCheckoutOptions): Promise<InternalCheckoutSelectors> {
+        if (typeof options?.fallback === 'function') {
```

Comment:
per our discussion, may be better to to remove this condition `==='functtion'` and refer only to the presence of fallback

Reply:
Thanks! 🙏 
Already updated